_config.yml

```diff
@@ -26,7 +26,7 @@ markdown: kramdown
 theme: minima
 plugins:
   - jekyll-feed
-gems: 
+plugins: 
```

Comment:
It would be better if you can have only one plugins field to avoid duplication.
Once you change it you can proceed.